src/deploy/functions/checkIam.ts

```diff
@@ -181,6 +262,9 @@ export async function ensureServiceAgentRoles(
     findRequiredBindings.push(service.requiredProjectBindings!(projectNumber, policy))
   );
   const allRequiredBindings = await Promise.all(findRequiredBindings);
+  allRequiredBindings.push(obtainPubSubServiceAgentBindings(projectNumber, policy));
+  allRequiredBindings.push(obtainDefaultComputeServiceAgentBindings(projectNumber, policy));
+  allRequiredBindings.push(obtainEventarcServiceAgentBindings(projectNumber, policy));
```

Comment:
IIUC, we try to setup these bindings on every deployment, regardless of whether the deployment requires it. Is that true? I'd like to avoid it, especially since I suspect CF3v1 customers don't even have eventarc enabled.

Reply:
Resolved over chat, we don't set these since we map to v2 services and v1 is ignored